pkg/vpcmodel/output.go

```diff
@@ -237,13 +240,19 @@ func WriteToFile(content, fileName string) (string, error) {
 	return content, nil
 }
 
-// getAsteriskDetails returns the info message about how non stateful conns are marked in the output, when relevant
-func getAsteriskDetails(uc OutputUseCase, hasStatelessConn bool, outFormat OutFormat) string {
+// getAsteriskDetails returns:
+// 1. the info message about how non stateful conns are marked in the output, when relevant
+// 2. the info message about how overAproximated conns are marked in the output, when relevant
```

Comment:
```suggestion
// getAsteriskDetails returns:
// 1. The info message regarding non stateful conns  in the output, when relevant
// 2. The info message regarding over-aproximated conns, when relevant
```

Reply:
fixed

Reply:
fixed spelling mistakes in the branch and pushed